### DISCUSSION TITLE
How to print output message from commands with plugin workspaces-tools

### DISCUSSION THREAD

**Discussion by nhattran998** [yarnpkg/berry#2907] [Q&A] [answered]
I use this command 
```
yarn workspaces foreach -Apv run dev",
```
But I don't see any output except when I press CTRL + C(out process) --> I can see the output.
Hope you guy can clear my case and thank you 😊

**Comment by merceyz** [comment, marked as answer]
By default the output is buffered, you can add the `-i` flag to print the output in real-time
https://yarnpkg.com/cli/workspaces/foreach#options-i%2C-interlaced

**Reply by nhattran998** [reply]
Oh, thank you for the quick answer!
